### PR TITLE
Feature flag showing org members

### PR DESCRIPTION
**Pull request by crabsinger**
### <samp>🤖 Generated by Copilot at 810e545</samp>

Add feature flag for organization view feature. Hide `Members` nav item unless the flag is enabled.

### CODE REVIEW

File: src/features/nav/NavCurrentOrg.tsx [hunk 1]
```diff
@@ -27,7 +28,7 @@ export const NavCurrentOrg = ({ org }: { org: NavOrg }) => {
         to={paths.vaultsForOrg(org.id)}
         icon={<DollarSign />}
       />
-      {isInOrg && (
+      {isFeatureEnabled('org_view') && isInOrg && (
```
Comment (CryptoGraffe) [inline src/features/nav/NavCurrentOrg.tsx:31]: Does this really work tho? where is the code that says "only show this if there are members" is it the isInOrg check?

**Comment by CryptoGraffe** [comment]
This is the problem tho , it won’t show up unless you already have org
members

**Comment by wingfeatherwave** [comment]
`Members` link in the current org menu is not working.  attempting to fix

**Review by wingfeatherwave — CHANGES_REQUESTED** [review]
`Members` link in the current org menu is not working.  attempting to fix

**Review by wingfeatherwave — APPROVED** [review]

**Pull request merged**